packages/expo-processing/index.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { GLView } from 'expo';
+
+const Browser = require('processing-js/lib/Browser');
+Browser.window = window;
+const Processing = require('processing-js/src')(Browser);
+
+export class ProcessingView extends React.Component {
+  componentWillUnmount() {
+    if (this._p) {
+      this._p.exit();
+      this._p = null;
+    }
+  }
+
+  render() {
+    return (
+      <GLView
+        {...this.props}
```

Comment:
This needs to filter out the `sketch` prop

Reply:
Figured I would make another PR for that, for the version bump